pkg/time/time_skew_amd64.go

```diff
@@ -41,14 +45,14 @@ const (
 	externVarTvNsecDelta  = "TV_NSEC_DELTA"
 )
 
-type TimeSkew struct {
+type SkewClockGetTime struct {
 	deltaSeconds     int64
 	deltaNanoSeconds int64
 	clockIDsMask     uint64
 	fakeImage        *FakeImage
 }
 
-func NewTimeSkew(deltaSeconds int64, deltaNanoSeconds int64, clockIDsMask uint64) (*TimeSkew, error) {
+func NewTimeSkew(deltaSeconds int64, deltaNanoSeconds int64, clockIDsMask uint64) (*SkewClockGetTime, error) {
```

Comment:
```suggestion
func NewSkewClockGetTime(deltaSeconds int64, deltaNanoSeconds int64, clockIDsMask uint64) (*SkewClockGetTime, error) {
```

please use the name of the struct as constructor name.